ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift

```diff
@@ -766,11 +766,22 @@ class SourcepointClientCoordinator: SPClientCoordinator {
             state.ccpa?.status = ccpa.status
             state.ccpa?.GPPData = ccpa.GPPData
         }
+        if let usnat = response.usnat, campaign == .usnat {
+            state.usnat?.dateCreated = usnat.dateCreated
+            state.usnat?.expirationDate = usnat.expirationDate
+            state.usnat?.consentStatus = usnat.consentStatus
+            state.usnat?.GPPData = usnat.GPPData
+            // state.usnat?.categories = usnat.сategories
+            // state.usnat?.consentStrings = usnat.consentStrings
+            // state.usnat?.statuses.consentedToAll = usnat.consentedToAll
+            // state.usnat?.consentStatus.rejectedAny = usnat.rejectedAny
+            // state.usnat?.statuses.gpcStatus = usnat.gpcEnabled
```

Comment:
Hey @andresilveirah ! 
This (commented) data is not saved for ccpa campaign, however, it is present in usnat. 
Do you think I should save it to storage?

Reply:
yes sir

Reply:
Done!